gatling-core/src/main/scala/io/gatling/core/json/Json.scala

```diff
@@ -21,10 +21,14 @@ import scala.collection.JavaConverters._
 
 import com.dongxiguo.fastring.Fastring.Implicits._
 
+import com.fasterxml.jackson.databind.ObjectMapper
+
 object Json {
 
   def stringify(value: Any, isRootObject: Boolean = true): String =
     fastringify(value, isRootObject).toString()
+  
+  private val objectMapper = new ObjectMapper
```

Comment:
I agree, The alternative would have been to create the mapper for the instance. I will try and fix this.